### DISCUSSION TITLE
Rename the report node a test which is paramterized

### DISCUSSION THREAD

**Discussion by sankethkamathk** [christiansandberg/pytest-reporter-html1#21] [General]
I have my test like this 
@pytest.mark.parametrize("data", load_device_deployment_data())
def test_device_negative_deployment(data):

and in the report, I get the name of tests as shown in the screenshot.. How do I override the data0,data1 to have a meaningful name 

![image](https://github.com/christiansandberg/pytest-reporter-html1/assets/97147710/906c71b1-bd4c-4d6a-8838-f0b626248980)

**Comment by christiansandberg** [comment]
You can use ids argument to generate a better name.

https://docs.pytest.org/en/7.4.x/example/parametrize.html#different-options-for-test-ids

https://docs.pytest.org/en/7.4.x/reference/reference.html#pytest.Metafunc.parametrize
